src/bento/components/bento-date-countdown/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'choumx'}, {name: 'ampproject/wg-components'}],
```

Comment:
FMI who is `choumx`? Do we still need to add them?

Reply:
Idk this file is just copied from `extensions/amp-date-countdown/OWNERS/` I'm tagging @choumx so they at least know this is happening.